webpack.config.js

```diff
@@ -0,0 +1,60 @@
+var Encore = require('@symfony/webpack-encore');
+
+Encore
+    // directory where compiled assets will be stored
+    .setOutputPath('public/build/')
+    // public path used by the web server to access the output path
+    .setPublicPath('/build')
+    // only needed for CDN's or sub-directory deploy
+    //.setManifestKeyPrefix('build/')
+
+    /*
+     * ENTRY CONFIG
+     *
+     * Add 1 entry for each "page" of your app
+     * (including one that's included on every page - e.g. "app")
+     *
+     * Each entry will result in one JavaScript file (e.g. app.js)
+     * and one CSS file (e.g. app.scss) if you JavaScript imports CSS.
+     */
+    .addEntry('app-dark', './assets/js/app-dark.js')
+    .addEntry('app-light', './assets/js/app-light.js')
```

Comment:
This is going to bundle two versions of the JS too - might be better to bundle everything together and determine which theme is active by adding a class to the body element dynamically. The theme customisations can then be scoped under that class.